web/src/components/identity/Access.tsx

```diff
@@ -1,34 +1,33 @@
-import React, { Component, Fragment } from "react";
-import { withRouter, Switch, Route } from "react-router-dom";
+import React, { useEffect } from "react";
+import { useHistory } from "react-router-dom";
 
-import withTheme from "@src/components/context/withTheme";
 // import NotFound from "../static/NotFound";
 // import SubNavBar from "@src/components/shared/SubNavBar";
 // import ConfigureIngress from "@src/components/identity/ConfigureIngress";
 import IdentityProviders from "@src/components/identity/IdentityProviders";
 
 import "@src/scss/components/identity/IdentityManagement.scss";
 
-class Access extends Component {
-  componentDidMount() {
-    const { history } = this.props;
+type Props = {
+  isKurlEnabled: boolean;
+  isGeoaxisSupported: boolean;
+};
 
+const Access = (props: Props) => {
+  const history = useHistory();
+  // TODO: move this into a redirect route or update links to default to /identity-providers
```

Comment:
don't want to deal with this until we clean up the routes and put them into their own files